samples/ControlCatalog/App.xaml.cs

```diff
@@ -11,25 +11,17 @@ public class App : Application
     {
         public static Styles FluentDark = new Styles
         {
-            new StyleInclude(new Uri("resm:Styles?assembly=ControlCatalog"))
-            {
-                Source = new Uri("resm:Avalonia.Themes.Default.Accents.BaseDark.xaml?assembly=Avalonia.Themes.Default")
-            },
-            new StyleInclude(new Uri("resm:Styles?assembly=ControlCatalog"))
+            new StyleInclude(new Uri("avares://ControlCatalog/Styles"))
```

Comment:
@danwalmsley  do we still have other resm stuff that can be replaced with avares?